src/engine-actions/create-wallet.js

```diff
@@ -7,7 +7,8 @@ const {
  * Creates a wallet
  *
  * @param {string} password - wallet password, used to unlock lnd wallet
- * @returns {Array<string>} 24 word cipher seed mnemonic
+ * @returns {Promise<Array<string>>} 24 word cipher seed mnemonic
+ * @this {{walletUnlocker: import('../lnd-setup').LndWalletUnlockerClient}}
```

Comment:
Is this the best way to do this?

Reply:
I replaced all the inline imports with typedefs to make it cleaner